helm/extendedapi/values.yaml

```diff
@@ -1,7 +1,7 @@
 image:
   repository: quay.io/domino
   container: extendedapi
-  appVersion: v0.0.2
+  appVersion: domsed
```

Comment:
Can you provide a version numb v1.0.0 (We can add in the release notes that we added here)